common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4376,6 +4395,78 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("120s")
 
+  val DYNAMIC_CONFIG_STORE_DB_DRIVER_CLASS_NAME: ConfigEntry[String] =
+    buildConf("celeborn.dynamicConfig.store.db.driverClassName")
+      .categories("master", "worker")
+      .version("0.5.0")
+      .doc("The jdbc driver class name of db store backend.")
+      .stringConf
+      .createWithDefaultString("")
+
+  val DYNAMIC_CONFIG_STORE_DB_JDBC_URL: ConfigEntry[String] =
+    buildConf("celeborn.dynamicConfig.store.db.jdbcUrl")
+      .categories("master", "worker")
+      .version("0.5.0")
+      .doc("The jdbc url of db store backend.")
+      .stringConf
+      .createWithDefaultString("")
+
+  val DYNAMIC_CONFIG_STORE_DB_USERNAME: ConfigEntry[String] =
+    buildConf("celeborn.dynamicConfig.store.db.username")
+      .categories("master", "worker")
+      .version("0.5.0")
+      .doc("The username of db store backend.")
+      .stringConf
+      .createWithDefaultString("")
+
+  val DYNAMIC_CONFIG_STORE_DB_PASSWORD: ConfigEntry[String] =
+    buildConf("celeborn.dynamicConfig.store.db.password")
+      .categories("master", "worker")
+      .version("0.5.0")
+      .doc("The password of db store backend.")
+      .stringConf
+      .createWithDefaultString("")
+
+  val DYNAMIC_CONFIG_STORE_DB_CONNECTION_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.dynamicConfig.store.db.connectionTimeout")
+      .categories("master", "worker")
+      .version("0.5.0")
+      .doc("The connection timeout that a client will wait for a connection from the pool for db store backend.")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("30s")
+
+  val DYNAMIC_CONFIG_STORE_DB_IDLE_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.dynamicConfig.store.db.idleTimeout")
```

Comment:
a leason we get from handling ratis configurations, it's better to use a prefix to passthrough all underlying(in this case, hikari) configurations.

Reply:
Done

Reply:
I mean passing ALL configurations starting with a prefix to the underlying Hikari pool, then we don't need to listing all keys in CelebornConf